hikey.mk

```diff
@@ -292,20 +292,28 @@ grub-flags := CC="$(CCACHE)gcc" \
 	TARGET_RANLIB="$(AARCH64_CROSS_COMPILE)ranlib" \
 	TARGET_STRIP="$(AARCH64_CROSS_COMPILE)strip"
 
+GRUB_MODULES += boot chain configfile echo efinet eval ext2 fat font gettext \
+                gfxterm gzio help linux loadenv lsefi normal part_gpt \
+                part_msdos read regexp search search_fs_file search_fs_uuid \
+                search_label terminal terminfo test tftp time
```

Comment:
You are adding more modules here, are they all needed?

Reply:
We've had this discussion before. : P
https://github.com/OP-TEE/build/pull/154#discussion_r121326578

Reply:
Not applicable, this is hikey.mk not hikey_debian.mk, we have a very simple, limited environment and I bet you a beer that not all modules are useful. Can you tell, for each of them, what it used for and/or what error happens if removed?

Reply:
Ok. I'll revert back to the original list of modules.